solutions/client-landing-zone/setters.yaml

```diff
@@ -170,6 +176,10 @@ data:
   #
   # dns project id created during core-landing-zone package deployment
   # customization: required, obtain value from the core-landing-zone setters.yaml
+  core-dns-project-id: dns-project-12345
```

Comment:
should we add core in front of dns-project-12345 -> core-dns-project-12345?

Reply:
good idea
update pushed